data/campaigns/Dead_Water/units/Child_King.cfg

```diff
@@ -14,7 +14,7 @@
     advances_to=Merman Young King
     cost=8
     usage=fighter
-    description= _ "When a mer king dies, the next in line becomes king however young he is. It is his job to lead his troops in combat if necessary. Good soldiers will do as they are told, but a very young king will get little real respect."
+    description= _ "When a merman king dies, the next in line becomes king however young he is. It is his job to lead his troops in combat if necessary. Good soldiers will do as they are told, but a very young king will get little real respect."
```

Comment:
This is pretty subjective, but I'm fine with just "mer" in this case.

Reply:
Matches the campaign description:
![image](https://user-images.githubusercontent.com/41433185/135739193-a493e667-01e5-485e-8f25-c73c3dced72a.png)

Reply:
hm, seeing "mer-sorceress" in there makes me wonder if we can replace the space with a hyphen to make it "mer-king"?

Reply:
Agree, either leave the original or just hyphenate it.